src/components/places-list/places-list.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import { Offer } from '../../types/offer';
+import PlacesCard from '../places-card/places-card';
+import { Nullable } from 'vitest';
+
+type PlacesListProps = {
+  offers: Offer[];
+};
+
+function PlacesList({ offers }: PlacesListProps): JSX.Element {
+  const [activeOffer, setActiveOffer] = useState<Nullable<Offer>>(null);
```

Comment:
Если параметр не нужен можно его опустить
```suggestion
  const [_, setActiveOffer] = useState<Nullable<Offer>>(null);
```